### ISSUE TITLE
Bug? Problem with spatial searches

### ISSUE THREAD

**Issue by Wiscmapper** [geoblacklight/geoblacklight#827]
I'm noticing odd results when performing spatial searches.  I'll re-create using the BTAA geoportal at https://geo.btaa.org/.

1. Using the map (left side of screen) on the main page, zoom in to a small piece of geography... perhaps neighborhood level.

2. Click the blue "search here" button.  In my example, 2,290 records are returned.  (Make sure "map view" is selected) 

![image](https://user-images.githubusercontent.com/3149952/62245706-c3f47900-b3a7-11e9-8334-532e74dc4931.png)

3.  Look at the results near the end of the list... page 115 in my example.  In this case, there will be many records for Penn State showing up in the Madison area.  True, Penn State could have records covering Wisconsin... BUT why does the "Water Quality - Central Appalachian Forest Ecoregion" show up in Madison?  It is not a matter of an incorrect bounding box.  This record from Penn State looks exactly as I would expect (covers Pennsylvania only), but yet it shows up in a spatial search over Madison: https://geo.btaa.org/catalog/668651a3-6027-49a1-b4dd-c526f46f6626

Another example:
![image](https://user-images.githubusercontent.com/3149952/62249621-74667b00-b3b0-11e9-8d90-a8aea167904c.png)

@karenmajewicz Do you see the same behavior?

**Comment by karenmajewicz** [comment]
@Wiscmapper I do see this behavior. We'll look into it.

**Comment by Wiscmapper** [comment]
@karenmajewicz Okay, more information.  I found a major screw-up in the GBL records that I was generating via an XSLT.  Basically, I had east and west lon reversed!  Once I fixed that, the search results are much more in line with what I would expect.  

Therefore, I highly doubt there is a bug in the software.

BUT I'm holding off closing this issue for the time being... perhaps there is a similar QA issue in solr_geom for some records in the BTAA portal?
